tests/common.rs

```diff
@@ -38,45 +38,34 @@ pub async fn rustls_type(s: &str) -> Result<Client<hyper_rustls::HttpsConnector<
                     }
             });
             caps.insert("goog:chromeOptions".to_string(), opts.clone());
-            ClientBuilder::rustls().capabilities(caps).connect("http://localhost:9515").await
+            caps
         }
         browser => unimplemented!("unsupported browser backend {}", browser),
     }
 }
 
+#[cfg(feature = "rustls-tls")]
+pub async fn rustls_type(s: &str) -> Result<Client<hyper_rustls::HttpsConnector<hyper::client::HttpConnector>>, error::NewSessionError> {
+    let caps = make_capabilities(s);
+    let url = match s {
```

Comment:
This `match` should also be factored out so we don't have to repeat it for each backend type.